pkg/build/controller/strategy/sti.go

```diff
@@ -48,6 +52,11 @@ func (bs *SourceBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod,
 		{Name: "BUILD_LOGLEVEL", Value: fmt.Sprintf("%d", cmdutil.GetLogLevel())},
 	}
 
+	// check if can run container as root
+	if !bs.canRunAsRoot(build) {
+		containerEnv = append(containerEnv, kapi.EnvVar{Name: "ALLOWED_UIDS", Value: "1-"})
+	}
```

Comment:
both get sent to the build container. From what I've seen so far, the container ends up with the last value specified in the list.